PopPUNK/network.py

```diff
@@ -87,7 +87,7 @@ def fetchNetwork(network_dir, model, refList, ref_graph = False,
     sys.stderr.write("Network loaded: " + str(len(list(genomeNetwork.vertices()))) + " samples\n")
 
     # Ensure all in dists are in final network
-    networkMissing = set(range(len(refList))).difference(list(genomeNetwork.vertices()))
+    networkMissing = set(map(str,set(range(len(refList))).difference(list(genomeNetwork.vertices()))))
```

Comment:
Can you check if this code for `networkMissing` is used anywhere else, and also make this update there, if necessary?